### ISSUE TITLE
Getting Run Time Error!

### ISSUE THREAD

**Issue by Krishnasony** [appspell/Wild-Scroll#1]
java.lang.NoClassDefFoundError: Failed resolution of: Lkotlinx/coroutines/experimental/BuildersKt;

**Comment by Krishnasony** [comment]
Please help me to solve this issue

**Comment by chitgoks** [comment]
yep same here. this project looks dead.

**Comment by appspell** [comment]
Yes, unfortunately, you're right. It's easy to fix bugs (if you need it) but to continue developing the project requires refactoring.
